### ISSUE TITLE
[Bug] API - api.stocks.fa.models.polygon.get_financials - Headers return "Value" instead of "Date"

### ISSUE THREAD

**Issue by deeleeramone** [OpenBB-finance/OpenBBTerminal#2512]
To recreate: 
```
api.stocks.fa.models.polygon.get_financials(symbol = "MSFT", statement= 'cash')
```
![Screen Shot 2022-09-07 at 8 14 10 AM](https://user-images.githubusercontent.com/85772166/188915748-cc4b7774-9435-4f0d-8004-1ba9db292b53.png)

**Comment by deeleeramone** [comment]
![Screen Shot 2022-09-07 at 8 36 00 AM](https://user-images.githubusercontent.com/85772166/188919671-6111f887-00fd-436b-9758-7e0e6d5a5e1d.png)

There is also more data from the source which can be extracted for Fundamentals. There is a total of six groupings of data. 

Balance Sheet
Cash Flow Statement
Income Statement
Comprehensive Income
General
Key Ratios

It follows the XBRL convention:

http://www.xbrlsite.com/2016/fac/v3/Documentation/FundamentalAccountingConceptsList.html

**Comment by hjoaquim** [comment]
Will be fixed by #2349 

![image](https://user-images.githubusercontent.com/48914296/190867978-f60c1bf7-93aa-4c2c-80da-bf9fff165ee1.png)
